app/assets/stylesheets/application.scss

```diff
@@ -312,10 +326,17 @@ main {
           display: inline-block;
         }
 
-        a {
+        button {
           @include core-16;
           padding: 0  5px 0 5px;
           display: inline-block;
+          -webkit-font-smoothing: antialiased;
+          -moz-osx-font-smoothing: grayscale;
```

Comment:
I'm not sure where GOV.UK stands on vendor prefixes - will leave that to someone else to confirm (I'd like to know too!) 😄

Reply:
going by the new typography styles in [govuk-frontend ](https://github.com/alphagov/govuk-frontend/blob/master/src/helpers/_typography.scss#L15)

I know this app is still using the (now deprecated) govuk_frontend_toolkit. I can remove the added prefixes, no problem